helm/ovn-kubernetes/templates/ovnkube-monitor.yaml

```diff
@@ -6,6 +6,9 @@ kind: ServiceMonitor
 metadata:
   labels:
     k8s-app: ovnkube-master
+    {{- with .Values.monitoring.serviceMonitorSelectorLabels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
```

Comment:
@csahithi can you please make the changes per our downstream code?

```
metadata:
  {{- with .Values.monitoring.serviceMonitorSelectorLabels }}
  labels:
    {{- toYaml . | nindent 4 }}
  {{- end }}
```

Reply:
move ` k8s-app: ovnkube-master` label to values.yaml along with the other label

Reply:
@girishmg updated the value to `commonServiceMonitorSelectorLabels`